tests/unit/record.spec.js

```diff
@@ -1,12 +1,5 @@
-import { shallowMount } from "@vue/test-utils";
-import Record from "@/components/Record.vue";
-
 describe("Record.vue", () => {
-  it("renders data.record when passed", () => {
-    const record = "new message";
-    const wrapper = shallowMount(Record, {
-      props: { modelValue: record }
-    });
-    expect(wrapper.text()).toMatch(record);
+  it("renders title from result data", () => {
+    console.log("TODO");
```

Comment:
I'd prefer a less noisy reminder here but I'm gonna agree this is fine for now lol :)